skyportal/models.py

```diff
@@ -314,10 +314,10 @@ class SourceView(Base):
 class Telescope(Base):
     name = sa.Column(sa.String, nullable=False)
     nickname = sa.Column(sa.String, nullable=False)
-    lat = sa.Column(sa.Float, nullable=False)
-    lon = sa.Column(sa.Float, nullable=False)
-    elevation = sa.Column(sa.Float, nullable=False)
-    diameter = sa.Column(sa.Float, nullable=False)
+    lat = sa.Column(sa.Float, nullable=True)
+    lon = sa.Column(sa.Float, nullable=True)
+    elevation = sa.Column(sa.Float, nullable=True)
```

Comment:
^^ These three must not be null in order to do observability calculations. 

Diameter being null is fine

Reply:
I had to do this since you put in HST!

Reply:
Good point... OK. I will account for this in an upcoming observability PR